### PR TITLE
Add Blazor Spreadsheet to Community page

### LINKED ISSUE #372: Add new sample (Blazor Excel)
If you like please add this to samples list.

https://lupblazordemos.z13.web.core.windows.net/ExcelPage

https://github.com/Lupusa87/BlazorExcel

untitled

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #372

### CODE REVIEW

File: community.md [hunk 1]
```diff
@@ -81,6 +81,7 @@ Below is a list of projects and resources from the Blazor community (links in ea
 * [Blazor Password Pattern](https://github.com/Lupusa87/BlazorPasswordPattern) ([Demo](https://lupusassblazordemos.azurewebsites.net)) &ndash; Password pattern based on SVG.
 * [Blazor RealWorld example app](https://github.com/torhovland/blazor-realworld-example-app) &ndash; A full-stack app built with Blazor that includes CRUD operations, authentication, routing, and pagination.
 * [Blazor RSS Reader](https://github.com/lohithgn/blazor-rss-reader) ([Demo](https://blazorrssreader.surge.sh/)) &ndash; RSS Reader using Blazor.
+* [Blazor Spreadsheet](https://github.com/Lupusa87/BlazorSpreadsheet) ([Demo](https://lupblazordemos.z13.web.core.windows.net/SpreadsheetPage)) &ndash; Spreadsheet app developed in Blazor.
```
Comment (danroth27) [inline community.md]: @lupusa87 Looks cool! Could you also add some text to the readme in the [BlazorSpreadsheet](https://github.com/Lupusa87/BlazorSpreadsheet) repo so folks know what the sample is for and how to run it?

**Comment by guardrex** [comment]
@Lupusa87 Ping me back here when you get the README done.

**Comment by Lupusa87** [comment]
@guardrex I did some updates https://github.com/Lupusa87/BlazorSpreadsheet/blob/master/README.md

**Comment by guardrex** [comment]
Thanks @Lupusa87! :rocket:

**Pull request merged**